tests/unit/common.py

```diff
@@ -34,23 +35,36 @@ def get_master_port():
     return master_port
 
 
-def set_cuda_visibile():
+def set_accelerator_visibile():
```

Comment:
Please help fix this preexisting typo: visibile -> visible

Reply:
This is fixed.